app/controllers/application_controller.rb

```diff
@@ -16,7 +16,7 @@ def track_validation_error(form)
       form_object: form.class.name,
       request_path: request.path,
       user: current_user,
-      details: form.errors.messages.map { |field, messages| [field, { messages: messages, value: form.public_send(field) }] }.to_h,
+      details: form.errors.messages.map { |field, messages| [field, { messages: messages, value: (form.public_send(field) if form.respond_to?(field)) }] }.to_h,
```

Comment:
The key we get for these validation errors does not correspond to a model attribute. For example, we may get `:make_decisions` but the model has `:training_provider_can_make_decisions` and `:ratifying_provider_can_make_decisions` instead. Registering `nil` for value is probably ok, as there is enough context in the `ValidationError` to understand what is going on.

```
=> #<ValidationError:0x000055e2b1e30690
 id: 1,
 form_object: "ProviderRelationshipPermissions",
 user_id: 2,
 user_type: "ProviderUser",
 request_path: "/provider/organisation-settings/organisation-permissions/4",
 details:
  {"view_diversity_information"=>
    {"value"=>nil,
     "messages"=>["Select which organisations can view diversity information"]},
   "view_safeguarding_information"=>
    {"value"=>nil,
     "messages"=>["Select which organisations can view safeguarding information"]}},
 created_at: Tue, 13 Jul 2021 14:53:34.885582000 BST +01:00,
 updated_at: Tue, 13 Jul 2021 14:53:34.885582000 BST +01:00,
 service: "manage">
```